atlas/additional-guides/troubleshooting-builds.mdx

```diff
@@ -67,6 +67,13 @@ After the platform pulls your code and runs `npm install`, it will execute the `
   },
 ```
 
+### In case the process of creating an application from an Atlas Blueprint fails
+
+1. If you selected to create a repository within an organisation, make sure the GitHub user you're authenticated as has enough permissions to create new repositories in this GitHub org.
+2. For Blueprints to work properly, we - for now - require your default branch to be called 'main'. You can change this setting globally for all new repositories in your ![GitHub repository settings](https://github.com/settings/repositories).
+3. <img width="626" alt="image" src="https://user-images.githubusercontent.com/15729608/178017600-27e4a0cd-14e1-4312-8489-7d424bec26d7.png">
```

Comment:
Same image issue I've mentioned in the last few PRs ew: images; I'd also make sure that the image is below the ordered list here